lib/Ohmbrewer_Thermostat.cpp

```diff
@@ -383,8 +439,11 @@ int Ohmbrewer::Thermostat::doUpdate(String &args, Ohmbrewer::Equipment::args_map
  * @returns The list of physical pins that the Thermostat is connected to.
  */
 std::list<int>* Ohmbrewer::Thermostat::whichPins() const {
-    // TODO: Implement Thermostat::whichPins
-    //return {&_pins, &getElement(), &getSensor()};
-    return _pins;
+    //return {temp buPin, element powerPin, element controlPin};
+    std::list<int>* pins = new std::list<int>();
+    pins->push_back(_tempSensor->getBusPin());
+    pins->push_back(_heatingElm->getPowerPin());
+    pins->push_back(_heatingElm->getControlPin());
+    return pins;
```

Comment:
Instead of manually specifying the pins for the heating element, just concatenate the list to what you've pushed into pins.
See http://stackoverflow.com/questions/1449703/how-to-append-a-listt-object-to-another for details.